website/docs/interactors/4-matchers-assertions.md

```diff
@@ -0,0 +1,228 @@
+---
+id: assertions-matchers
+title: Assertions and Matchers
+---
+
+This page will discuss the different ways you can assert with interactors. Afterwards, we will do a deep dive on how you can utilize matchers that are offered out-of-the-box with BigTest, and also go over how you can compose your own reusable matchers.
+
+## Assertions
+In the [Quick Start](/docs/interactors/#making-test-assertions) we briefly touched on the assertion methods that are available for all interactors - `exists()` and `absent()`. There is also `has()` which allows you pass in a filter as its argument.
+
+:::note
+These different assertion methods are equivalents of Jest's `expect` and Cypress' `should`. When refactoring your test with Interactors, you would replace those constructs with the interactors' assertion methods.
+:::
+
+In the case of a form with textfields, you would assert the placeholder against a textfield like this:
+
+```js
+TextField({ id: 'username-id' }).has({ placeholder: 'USERNAME' });
+```
+
+The difference between this approach and using `exists()` is that `exists()` is less refined as it will succeed as long as there is at least one match. With the two textfields for the username and password, the assertion would fail on if we were to write a test like this:
+
+```js
+TextField().has({ placeholder: 'USERNAME' });
+```
+
+It would fail on account of the textfield that has the placeholder value `PASSWORD`. You therefore need to choose the assertion method that is most appropriate for your tests.
```

Comment:
> You therefore need to choose the assertion method that is most appropriate for your tests.

Instead of listing caveats of each method, can we turn it around and list the use cases and discuss which methods are the appropriate ones for each case?

Reply:
We updated this to show the positive advice about when to use each method first, and listing caveats second.